### DISCUSSION TITLE
"View in your space" button overlaying photo button in Sceneviewer

### DISCUSSION THREAD

**Discussion by SGECZ** [google/model-viewer#2762] [Q&A]
Hi there! Anyone an idea what causes this? Came back from vacation and it is like this suddenly - and we didn't change anything...

![get_attachment_url](https://user-images.githubusercontent.com/68609303/132317522-01388dd8-ee32-4829-b011-d64a0dea548b.jpg)

And why is that button even there - I am already in "my space".

thank you!

**Comment by elalish** [comment]
@tpsiaki This looks like a SceneViewer issue? @SGECZ can  you give us a repro link? And some repro steps, device/OS, etc?

**Reply by SGECZ** [reply]
> @tpsiaki This looks like a SceneViewer issue? @SGECZ can you give us a repro link? And some repro steps, device/OS, etc?

sure - you can use this link: https://cldxr.to/i6ojtdhe - just use the Start AR button once loaded 

It must have come with the recent update (which on my phone shows to be from September 3rd). 

And I tried it on 2 Samsungs - Note 10 and S9 - same thing

Thanks!

**Reply by eugenecys** [reply]
@SGECZ Can you reliably repro the issue? Can I trouble you to record the process of launching the asset from the intent till you see the issue, and also sending the app to the background and then reopening from recents screen? 

I can't repro the issue, even on an S9, so if you can link us to a video of the repro as outlined above it will be immensely helpful!

**Reply by SGECZ** [reply]
@eugenecys  Sure - please have a look here:

https://user-images.githubusercontent.com/68609303/132470841-bb7a2914-1a62-403c-a20b-1b67a55a3a1b.mp4

One more thing: When I try on my S9 (which I use only for testing), I am almost getting random results.

Like this one - never saw the "tap the arrow..." before
![Screenshot_20210908-102456_Google](https://user-images.githubusercontent.com/68609303/132474780-6749e4fd-e2ae-4267-aa87-e9f32f43d51f.jpg)

Or this one - where it also asked me to look for the wall
![Screenshot_20210908-102602_Google](https://user-images.githubusercontent.com/68609303/132474971-8f7b97e3-4482-4e28-af11-5d1ff55e9e01.jpg)

And here's the help button on top of the other button
![Screenshot_20210908-102519_Google](https://user-images.githubusercontent.com/68609303/132475083-a1738dce-7003-48ae-a9c4-b81a65e9ebeb.jpg)

All this is happening under the same URL: https://cldxr.to/i6ojtdhe 

With the same call to scene viewer every time.

**Reply by julienbfabre** [reply]
Just to chime in, i'm getting the same problem on my One Plus 6T.
The "view in your space" button starts off enlarged, clicking on it goes into regular model viewing (button remains enlarged) and clicking on it again goes back to AR and button finally shrinks to the side.

As additional info, this is how we are starting the AR:

```
intent://arvr.google.com/scene-viewer/1.0?file=https://cdn.cldxr.com/model3d/asset/604a7874c9508e4eeb28b08a/Ted_animated.glb&disable_occlusion=true&mode=ar_only&title=Ted&link=https%3A%2F%2Fcldxr.com#Intent;scheme=https;package=com.google.ar.core;action=android.intent.action.VIEW;S.browser_fallback_url=https://developers.google.com/ar;end;
```

Is there anything we should be changing in that intent link?

**Reply by eugenecys** [reply]
That intent link looks fine to me, this is most likely a bug on our side - I'll take a look at it. I'll let you know once we have something

**Comment by SGECZ** [comment]
I saw there was an update of the Google app today, so I had hope that this got fixed.

But no. :( 

@elalish @eugenecys - any news on this?

**Reply by eugenecys** [reply]
@SGECZ Can you check which version of the Google App did it update to? I made a change that went in with the 12.37 release to hopefully fix it, if it didn't let me know and I'll look into it again (I'm making a blind fix here as I can't get a repro on any of our devices including an S9).

**Reply by SGECZ** [reply]
@eugenecys - mine is 12.36 - update from Sept. 17th. So there is hope for the next one, I'll report back once I have that (not available for me yet).
